src/utils/builder/edit_member.rs

```diff
@@ -2,27 +2,33 @@ use serde_json::builder::ObjectBuilder;
 use std::default::Default;
 use ::model::{ChannelId, RoleId};
 
+/// A builder that edits properties of a ['Member'] via ['Context::edit_member'].
+
```

Comment:
Also, should link to the relevant structs. You can do this by changing the apostrophes to graves, and then linking at the bottom like:

```rs
///
/// [`Member`]: ../../model/struct.Member.html
```

paths are relative to the current file